api/src/pcapi/routes/backoffice_v3/templates/components/history/actions.html

```diff
@@ -56,6 +56,11 @@
                   </div>
                 {% endfor %}
               {% endif %}
+              {% set linked_incident_id = action.extraData.get('linked_incident_id', {}) %}
```

Comment:
Comme on attend un id, un dict comme valeur par défaut ne me semble pas cohérente, None suffira.
```suggestion
              {% set linked_incident_id = action.extraData.get('linked_incident_id') %}
```